app/controllers/analytics/districts_controller.rb

```diff
@@ -84,9 +84,21 @@ def set_dashboard_analytics(period, prev_periods)
   end
 
   def set_facility_keys
-    facilities = @organization_district.facilities.order(:name).pluck(:id, :name).to_h
-
-    @facility_keys = { total: "Total" }.merge(facilities).with_indifferent_access
+    district = {
+      id: :total,
+      name: "Total",
+      type: "District"
```

Comment:
![image](https://user-images.githubusercontent.com/16774200/84000383-259b5180-a982-11ea-8991-2c39013518fa.png)

The `Facility Type` : `District` column feels slightly confusing, can we maybe say **`Total - District`** and not have that column in the totals?

Reply:
I'll just stick a `nil` in there. The fact that it says `<District> Cohort Report` right above makes it clear what the total is 👍

Reply:
There was also a bug where the facilities' types weren't being printed in there. Fixed now.

Reply:
Ah, ok. That's what the empty cells were about. :shipit: